src/utils/transactions.ts

```diff
@@ -18,15 +18,14 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import {
   calcTxAmountsDeltaForAddress,
-  DUST_AMOUNT,
   getDirection,
   isConsolidationTx,
-  MIN_UTXO_SET_AMOUNT,
   TransactionDirection,
   TransactionInfoType
 } from '@alephium/sdk'
 import { AssetOutput, Output } from '@alephium/sdk/api/explorer'
 import { ALPH } from '@alephium/token-list'
+import { DUST_AMOUNT, MIN_UTXO_SET_AMOUNT } from '@alephium/web3'
```

Comment:
If we would have imported the DUST_AMOUNT from the web3 package we wouldn't have had this bug. But maybe it wasn't there when we first started working on this feature. In any case, I imported all constants from web3 now.